concrete/src/Http/Client/Client.php

```diff
@@ -45,6 +45,24 @@ public function setLogger(LoggerInterface $value = null)
         return $this;
     }
 
+    /**
+     * Send a request without following any redirects
+     *
+     * @see ZendClient::send()
+     * @param ZendRequest|null $request
+     * @return \Zend\Http\Response
+     */
+    public function sendWithoutRedirects(ZendRequest $request = null)
+    {
+        $maxRedirects = $this->config['maxredirects'];
```

Comment:
What about
```
$this->config['maxredirects'] = 0;
try {
    $result = $this->send($request);
} finally {
    $this->config['maxredirects'] = $maxRedirects;
}
```

that way we are sure we restore the previous value even in case of exceptions thrown...

Reply:
Very good point, updated.